dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBMongoDB/AzureCosmosDBMongoDBMemoryRecord.cs

```diff
@@ -12,6 +13,7 @@ namespace Microsoft.SemanticKernel.Connectors.AzureCosmosDBMongoDB;
 /// <summary>
 /// A MongoDB memory record.
 /// </summary>
+[Experimental("SKEXP0020")]
```

Comment:
I'm adding experimental to internal classes as well, since I don't want them accidentally used by us.